### ISSUE TITLE
Channels not playing

### ISSUE THREAD

**Issue by Taserwebs** [krreet/plugin.video.jiotv#131]
The channels in jio tv were not playing in kodi

**Comment by giridathac** [comment]
Yes, none of the channels are working. Pls let us know a workaround till the issue is fixed.

**Comment by Taserwebs** [comment]
Ok, rectify the issue soon, thankyou for your help

**Comment by ritik-gupta** [comment]
How did you rectify it @Taserwebs ?

**Comment by Taserwebs** [comment]
> How did you rectify it @Taserwebs ?

I don't know, i told the one who contributed this plugin to rectify it

**Comment by piscesbangalore** [comment]
Does anyone get login failed message?
I am using id and password option from jio fibre.

**Comment by Gokulakannanaj** [comment]
@Taserwebs Do you have any update on when it'll be resolved ? Or do you have any workaround?

**Comment by giridathac** [comment]
> Does anyone get login failed message? I am using id and password option from jio fibre.

Yes, before this disruption I did see failed login messages, but channels were playing . Now no channels are playing, accompanied with login errors.

**Comment by Taserwebs** [comment]
> @krreet @Taserwebs Do you have any update on when it'll be resolved ? Or do you have any workaround?

Dont ping me , i dont know, iam also the one who is using this plugin

**Comment by supercyen041** [comment]
use kodi nexus 20.5 with otp login only (don't use email login)... all channel working fine except sony channel

**Comment by Gokulakannanaj** [comment]
> > @krreet @Taserwebs Do you have any update on when it'll be resolved ? Or do you have any workaround?
> 
> Dont ping me , i dont know, iam also the one who is using this plugin

Do you know the meaning of forum? If you aren't ready to answer, you can simply skip it without responding

**Comment by balaajaychittemetty** [comment]
@all - what's happening with Kodi from last 2 days? It's not working. Please let us know the work around to continue using it back

**Comment by technofreak24** [comment]
Solution is given on https://github.com/krreet/plugin.video.jiotv/issues/132#issuecomment-2211020331

**Comment by Taserwebs** [comment]
> > > @krreet @Taserwebs Do you have any update on when it'll be resolved ? Or do you have any workaround?
> > 
> > 
> > Dont ping me , i dont know, iam also the one who is using this plugin
> 
> Do you know the meaning of forum? If you aren't ready to answer, you can simply skip it without responding

Then why you have taged me, i gave the response correctly right, i also know the meaning of the forum, if we are taged we should give proper response

**Comment by pasupuletinarasimha999** [comment]
> Solution is given on [#132 (comment)](https://github.com/krreet/plugin.video.jiotv/issues/132#issuecomment-2211020331)

It is mentioned that we need to use kodi nexus 20.5 but what about webos case

**Comment by pasupuletinarasimha999** [comment]
> Solution is given on [#132 (comment)](https://github.com/krreet/plugin.video.jiotv/issues/132#issuecomment-2211020331)

@technofreak24 Please tell me workaround for webos also

**Comment by technofreak24** [comment]
> @pasupuletinarasimha999 : Please tell me workaround for webos also

Narasimha brother, You can try FireTV Stick or Android Set-top box for WebOS devices (like LG TV) since there is only KODI Omega v21 on official KODI releases source on https://mirrors.kodi.tv/releases/webos/
